recipe-server/client/control_new/routes.js

```diff
@@ -22,16 +24,16 @@ const routes = {
         crumb: 'View Recipe',
       },
     },
-    '/extensions': {
+    '/extension': {
```

Comment:
Interesting. It may be worth submitting a patch for `redux-little-router`, but that's a greater discussion to have with the group.